testsuite/features/support/twopence_init.rb

```diff
@@ -72,6 +72,7 @@ def twopence_init(target)
   $opensuse154arm_ssh_minion = twopence_init("ssh:#{ENV['OPENSUSE154ARM_SSHMINION']}") if ENV['OPENSUSE154ARM_SSHMINION']
   $sle12sp5_buildhost = twopence_init("ssh:#{ENV['SLE12SP5_BUILDHOST']}") if ENV['SLE12SP5_BUILDHOST']
   $sle15sp4_buildhost = twopence_init("ssh:#{ENV['SLE15SP4_BUILDHOST']}") if ENV['SLE15SP4_BUILDHOST']
+  $monitoring_server = twopence_init("ssh:#{ENV['MONITORING_SERVER']}") if ENV['MONITORING_SERVER']
```

Comment:
Do you already adjusted the `.bashrc` on the controller in sumaform and added those environment variable?

Reply:
@Bischoff did! https://github.com/uyuni-project/sumaform/pull/1223

Reply:
Alright. All good then :wink: